### PR TITLE
[Quickfix] Remove overriding plastic sheet tag

### PR DESCRIPTION
**Pull request by HoofedEar**
## About the PR 

Plastic sheets were overriding their tags, thus being rejected by the lathe's whitelists

**Screenshots**

**Changelog**

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml [hunk 1]
```diff
@@ -107,9 +107,6 @@
   name: plastic
   suffix: Full
   components:
-  - type: Tag
-    tags:
-    - Plastic
```
Comment (ElectroJr) [inline Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml:112]: Is the plastic tag not used for anything? Why not just add the `SheetOtherBase` tags to this prototype?

Comment (HoofedEar) [reply Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml:112]: good point

File: Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml [hunk 2]
```diff
@@ -110,6 +110,7 @@
   - type: Tag
     tags:
     - Plastic
+    - Sheet
```
Comment (ElectroJr) [inline Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml:113]: also DroneUsable?

Comment (HoofedEar) [reply Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml:113]: gaming and coding lol

**Pull request merged**